internal/webserver/server.go

```diff
@@ -120,6 +127,44 @@ func (webserver *WebServer) listenAndServe(serviceTimeout time.Duration, errChan
 	}
 	addr := fmt.Sprintf("%s:%d", bindAddress, config.Service.Port)
 
+	var ln net.Listener
+	var err error
+	listenMode := strings.ToLower(config.Service.SecurityOptions[bscfg.SecurityModeKey])
+	switch listenMode {
+	case zerotrust.ZeroTrustMode:
+		ozUrl := config.Service.SecurityOptions["OpenZitiController"]
+
+		secretProvider := bootstrapContainer.SecretProviderExtFrom(webserver.dic.Get)
+		ozToken, jwtErr := secretProvider.GetSelfJWT()
+		if jwtErr != nil {
+			lc.Errorf("zero trust mode enabled, but could not load jwt: %v", jwtErr)
+			return
```

Comment:
Send error to error channel before return. Same for other errors and panic missed below.
```suggestion
			lc.Errorf("zero trust mode enabled, but could not load jwt: %v", jwtErr)
			errChannel <- jwtErr
			return
```